README.md

```diff
@@ -85,15 +85,13 @@ Paths (FILE / DIRECTORY) are absolute paths or relative paths to working directo
        Use directory DIRECTORY to store backups.
        If this option is not given, folder 'backups/' in script's directory is created and used.
 -p | --snap
-       Use this option, if you are running nextcloud-snap (https://github.com/nextcloud/nextcloud-snap).
-       calcardbackup will then use the in the snap package included cli utility 'nextcloud.mysql-client'
-       to read the needed values from the database. Note that in order for this to work, calcardbackup has
-       to be run with sudo (even running as root without sudo will fail!).
+       This option is mandatory if you are running nextcloud-snap (https://github.com/nextcloud/nextcloud-snap).
+       With this option, calcardbackup has to be run with sudo (even running as root without sudo will fail!).
 -r | --remove N
        Remove backups older than N days from backup folder (N needs to be a positive integer).
 -s | --selfsigned
        Needs to be given if certificate is selfsigned or for any other reason not trustful to curl.
-       This option might only be needed when using option '-g|--get-via-http'
+       This option is only used in combination with option '-g|--get-via-http'
```

Comment:
cURL is used to retrieve status.php and the vendor of the ownCloud/Nextcloud installation to perform some additional checks. If cURL can't access the URL due to an untrustful certificate, calcardbackup will run without executing these checks. This option is only mandatory when using option -g together with a sefsigned certificate.
```suggestion
       curl is being used to perform some additional checks, which won't be executed when using
       a selfsigned or for any other reason untrustful certificate. Use this option to tell curl to trust
       the certificate in any case.
       This option is only mandatory in combination with option '-g|--get-via-http' and selfsigned certificate.
```